

"""
Никогда не игнорируйте ошибки которые появляются в Exception,
даже если вы все протестировали, и в Exception по вашему мнению
ничего не попадет, то рано или поздно он все же может отработать,
и так как вы не сделали логирования, то эту ошибку вы просто пропустите.

Поэтому если вы решили все же использовать Exception, то как минимум
сделайте возможность сохранять исключения, которые появляются в процессе
работы программы.
"""


def download_yandex_source_code():
    try:
        # Предположим, что эта ошибка появилась
        # спустя месяц активной разработки
        temp = [1, 2, 3]
        result = temp[255]
    except Exception:
        # В итоге клиенты получают ошибки,
        # а мы ничего не знаем об этом
        pass


def main():
    download_yandex_source_code()


if __name__ == "__main__":
    main()